main/src/ca/uwaterloo/flix/api/lsp/provider/InlayHintProvider.scala

```diff
@@ -15,13 +15,33 @@
  */
 package ca.uwaterloo.flix.api.lsp.provider
 
-import ca.uwaterloo.flix.api.lsp.{InlayHint, Range}
+import ca.uwaterloo.flix.api.lsp.acceptors.{FileAcceptor}
+import ca.uwaterloo.flix.language.ast.shared.SymUse
+import ca.uwaterloo.flix.language.ast.TypedAst.{Expr, Root}
+import ca.uwaterloo.flix.api.lsp.{Consumer, InlayHint, InlayHintKind, Position, Range}
+import ca.uwaterloo.flix.api.lsp.Visitor
 
 object InlayHintProvider {
```

Comment:
Because this feature - at least to begin with - is for us, let us add a boolean:

private val EnableEffectHints: Boolean = false

and  then use that in the body of getInlayHints